akka-actor/src/main/scala/akka/actor/TypedActor.scala

```diff
@@ -128,10 +128,6 @@ object TypedActor extends ExtensionId[TypedActorExtension] with ExtensionIdProvi
   @silent("deprecated")
   def get(context: ActorContext): TypedActorFactory = apply(context)
 
-  @deprecated("Use 'akka.actor.typed' API.", since = "2.6.0")
-  @silent("deprecated")
-  override def apply(system: ActorSystem): TypedActorExtension = super.apply(system)
-
```

Comment:
The 2.5 signature returns `Extension` but couldn't get that to compile so reverted to having Scala generate it for us. I think this was only added to give the deprecation warning, which is also on the object, so not really needed anyway.